MailCore/Cache/MailboxManager/MailboxManager+DB.swift

```diff
@@ -0,0 +1,42 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2022 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+
+public extension MailboxManager {
```

Comment:
As `RealmAccessible` is related to DB, maybe we can conform to this protocol in this file?

Reply:
Yes I actualy created this file to put it here to begin with. There is an issue where if I put the conformance in an extension here, code outside core fails to access it. (you cannot declare extesions as public) so this is what I came with.